### DISCUSSION TITLE
Device not connected to the home network

### DISCUSSION THREAD

**Discussion by yotitote** [theyosh/TerrariumPI#772] [Q&A] [answered]
Hey, this may be a dumb question but is there a way to enter the raspberry pi/terrarium pi website when I'm using a device that's not connected to the home network but for example mobile network? (I'm a new owner of raspberry pi and I don't have much experience)

**Comment by theyosh** [comment]
That is possible. That is called 'port forwarding'. But that is not something simple. And also, this will make everybody able to access your TerrariumPI setup. 

https://www.youtube.com/watch?v=AFeye1zLpFo
https://www.lifewire.com/how-to-port-forward-4163829
https://portforward.com/router.htm

But this is outside the support scope for me. My software is compatible with port forwarding. But setting it up, depends largely on your internet connection router.

**Comment by yotitote** [comment, marked as answer]
Thanks for the answer.
